x/accesscontrol/keeper/params.go

```diff
@@ -0,0 +1,21 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/accesscontrol/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// GetParams returns the total set params.
+func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
+	k.paramSpace.GetParamSet(ctx, &params)
+	return params
+}
+
+// SetParams sets the total set of params.
+func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
+	k.paramSpace.SetParamSet(ctx, &params)
+	for _, resourceDepedencyMapping := range params.GetMessageDependencyMapping() {
+		k.SetResourceDepedencyMapping(ctx, resourceDepedencyMapping)
```

Comment:
I think we would just want to not have any dependency mapping in params since theyre already in the store under their own key